lib/cache_db.rb

```diff
@@ -234,23 +247,23 @@ def all
     end
   end
 
-  def needs_update? url
-    data = self.get url
+  def needs_update?(url)
+    data = get url
 
     return true if data.nil?
 
     return true if data["modified"].nil?
 
     # For Flickr wrong size stuff
-    return true if data["image"].nil? or data["image"].match /_q/
+    return true if data["image"].nil? || data["image"].match(/_q/)
 
     # ~10 days * a random float
     time = Time.parse(data["modified"])
-    return (Time.now - time) > (860000 * rand)
+    (Time.now - time) > (860_000 * rand)
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.